src/common/lib/util/logger.ts

```diff
@@ -45,10 +45,9 @@ const getDefaultLoggers = (): [Function, Function] => {
   let consoleLogger;
   let errorLogger;
 
-  /* Can't just check for console && console.log; fails in IE <=9 */
   if (
     (typeof Window === 'undefined' && typeof WorkerGlobalScope === 'undefined') /* node */ ||
-    typeof globalObject?.console?.log?.apply === 'function' /* sensible browsers */
+    typeof globalObject?.console?.log === 'function' /* sensible browsers */
```

Comment:
This code is a bit confusing after the deletion in b00fb7c82aa4facaea70ea413c08e937c844373a:

- what is a "sensible browser"?
- under what circumstances is the `else` triggered now?

Reply:
Agree, it looks more confusing now without special cases for IE.
I think whole `if` statement can be rewritten to just check for `typeof globalObject?.console?.log === 'function'` (without typeofs for Window and WorkerGlobalScope for node), and call it a `sensible environment` - meaning that any reasonable environment we expect ably-js to be running on should have a `console` object available.
WDYT?

> under what circumstances is the else triggered now?

Nothing comes to mind immediately, but it is a safety check that if an environment doesn't have `console` for some reason, we default to noop for log operations and don't fail whole application because of this.
Would like to remove it?